docs/index.rst

```diff
@@ -21,13 +21,13 @@ The hipBLAS documentation is structured as follows:
 
     * :ref:`install`
 
-  .. grid-item-card:: How-To
+  .. grid-item-card:: How to
```

Comment:
I liked it better hyphenated, but I'm not an English expert or anything so I could just be wrong haha

Reply:
Mine is not to question why...